frontend/packages/console-app/src/components/cloud-shell/cloud-shell-utils.ts

```diff
@@ -36,37 +55,85 @@ export const CLOUD_SHELL_PROTECTED_NAMESPACE = 'openshift-terminal';
 
 export const createCloudShellResourceName = () => `terminal-${getRandomChars(6)}`;
 
-export const newCloudShellWorkSpace = (name: string, namespace: string): CloudShellResource => ({
-  apiVersion: 'workspace.devfile.io/v1alpha1',
-  kind: 'DevWorkspace',
-  metadata: {
-    name,
-    namespace,
-    labels: {
-      [CLOUD_SHELL_LABEL]: 'true',
+const v1alpha1DevworkspaceComponent = () => {
+  return [
```

Comment:
no need for method, can be a constant
```suggestion
const v1alpha1DevworkspaceComponent = [
```